iam-role.tf

```diff
@@ -47,6 +47,35 @@ resource "aws_iam_role_policy_attachment" "xray" {
   role       = aws_iam_role.this[0].name
 }
 
+# Allow Lambda to access specific EFS"
+data "aws_iam_policy_document" "file_system" {
```

Comment:
Can you provide a `sid` here to make this more descriptive?

Reply:
Done.